### ISSUE TITLE
Opening menu option

### ISSUE THREAD

**Issue by Aeromancer** [We-the-People-civ4col-mod/Mod#489]
I changed the opening menu option in the We the People mod.   This has caused the game to be unable to load the mod.  

Uninstalling the mod and civ iv col both fail to fix the issue once they've both been installed from fresh.

**Comment by raystuttgart** [comment]
Please check the folders. **Uninstall often leasves back files** that you have put there manually.

**Comment by Aeromancer** [comment]
I'm running on steam so when I uninstalled I made sure to delete any remaining folder in the steam folder for Colonization as we as the folder in Documents\My Games\Civilization IV Colonization.  There isn't a third place that it creates a folder is there?  

I haven't added any music so the only music is either what comes with the game or with the mod. 

Also to make sure I've turned off cloud saving on steam in case that is causing the issue.
